client/extensions/woocommerce/state/sites/setup-choices/test/actions.js

```diff
@@ -76,4 +90,328 @@ describe( 'actions', () => {
+						finished_initial_install_of_required_plugins: false,
+						set_store_address_during_initial_setup: false,
+					}
+				} );
+			} );
+		} );
+	} );
+
+	describe( '#setFinishedInstallOfRequiredPlugins', () => {
+		const siteId = '123';
+
+		useSandbox();
+		useNock( ( nock ) => {
+			nock( 'https://public-api.wordpress.com:443' )
+				.persist()
+				.post( '/rest/v1.1/sites/123/calypso-preferences/woocommerce', {
+					finished_initial_install_of_required_plugins: true,
+				} )
+				.reply( 200, {
+					finished_initial_setup: false,
+					opted_out_of_shipping_setup: false,
+					opted_out_of_taxes_setup: false,
+					tried_customizer_during_initial_setup: false,
+					finished_initial_install_of_required_plugins: true,
+					set_store_address_during_initial_setup: false,
+				} );
+		} );
+
+		it( 'should dispatch an action', () => {
+			const getState = () => ( {} );
+			const dispatch = spy();
+			setFinishedInstallOfRequiredPlugins( siteId, true )( dispatch, getState );
+			expect( dispatch ).to.have.been.calledWith( {
+				type: WOOCOMMERCE_SETUP_CHOICE_UPDATE_REQUEST,
+				siteId,
+				key: 'finished_initial_install_of_required_plugins',
+				value: true
+			} );
+		} );
+
+		it( 'should dispatch a success action with setup choices when request completes', () => {
+			const getState = () => ( {} );
+			const dispatch = spy();
+			const response = setFinishedInstallOfRequiredPlugins( siteId, true )( dispatch, getState );
+
+			return response.then( () => {
+				expect( dispatch ).to.have.been.calledWith( {
+					type: WOOCOMMERCE_SETUP_CHOICE_UPDATE_REQUEST_SUCCESS,
+					siteId,
+					data: {
+						finished_initial_setup: false,
+						opted_out_of_shipping_setup: false,
+						opted_out_of_taxes_setup: false,
+						tried_customizer_during_initial_setup: false,
+						finished_initial_install_of_required_plugins: true,
+						set_store_address_during_initial_setup: false,
+					}
+				} );
+			} );
+		} );
+	} );
+
+	describe( '#setSetStoreAddressDuringInitialSetup', () => {
+		const siteId = '123';
+
+		useSandbox();
+		useNock( ( nock ) => {
+			nock( 'https://public-api.wordpress.com:443' )
+				.persist()
+				.post( '/rest/v1.1/sites/123/calypso-preferences/woocommerce', {
+					set_store_address_during_initial_setup: true,
+				} )
+				.reply( 200, {
+					finished_initial_setup: false,
+					opted_out_of_shipping_setup: false,
+					opted_out_of_taxes_setup: false,
+					tried_customizer_during_initial_setup: false,
+					finished_initial_install_of_required_plugins: false,
+					set_store_address_during_initial_setup: true,
+				} );
+		} );
+
+		it( 'should dispatch an action', () => {
+			const getState = () => ( {} );
+			const dispatch = spy();
+			setSetStoreAddressDuringInitialSetup( siteId, true )( dispatch, getState );
+			expect( dispatch ).to.have.been.calledWith( {
+				type: WOOCOMMERCE_SETUP_CHOICE_UPDATE_REQUEST,
+				siteId,
+				key: 'set_store_address_during_initial_setup',
+				value: true
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/15223/commits/004863a3c5f48ce79a2d0ea93ba459112536c873